r2/service.go

```diff
@@ -28,8 +28,11 @@ import (
 	"reflect"
 	"strconv"
 	"strings"
+	"sync"
 	"time"
 
+	"github.com/uber-go/tally"
```

Comment:
nit: convention is to include 3rd party imports at the bottom part of the import block.